src/components/commons/prismicTypes.ts

```diff
@@ -2,6 +2,9 @@ export interface Posts {
   title: {
     text: string;
   };
+  title_de: {
```

Comment:
is this always get returned? if not just to be safe
```suggestion
  title_de?: {
```

Reply:
okie, included!